Library/Homebrew/cmd/audit.rb

```diff
@@ -230,6 +227,12 @@ def audit_file
     if present.include?("bottle modifier") && present.include?("bottle block")
       problem "Should not have `bottle :unneeded/:disable` and `bottle do`"
     end
+
+    return unless @style
+
+    style_violations = nil
+    Homebrew.check_style(@formula.path) { |rslt| style_violations = rslt }
```

Comment:
Sorry; shorthand for "result". I'm still learning to remember to spell everything out for `brew` code.

xu-cheng's version didn't occur to me since I'm still new at Ruby. Switched to that.